CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Ansible Modules for HPE OneView Change Log
 
+## v5.7.0
```

Comment:
keep (unreleased) beside release version.

Reply:
I think the version should be v5.6.1

Reply:
yeah, sorry it should be v5.6.1

Reply:
Done!

Reply:
Still the version is not changed

Reply:
@VenkateshRavula Done